### ISSUE TITLE
Fix Hardcoded References in SRS

### ISSUE THREAD

**Issue by samm82** [smiths/caseStudies#31]
Lines 288-289 of [glassbr_srs.tex](https://github.com/smiths/caseStudies/blob/master/CaseStudies/glass/Documentation/SRS/glassbr_srs.tex) are missing a reference to another section: "...standards for constructions using glass from {[}4-6{]} in
Section. The users of GlassBR can have a lower level of..."

# TODO
- [ ] Fix in caseStudies
- [ ] Fix in Drasil
  - [ ] Characteristics of Intended Reader
  - [ ] Terminology and Definitions
    - [ ] Glass Type
    - [ ] Load
  - [ ] Calculation of Capacity IM

**Comment by smiths** [comment]
The original text was wrong.  I've fixed it in 65bba6a.  The original intention would have been to use citations, but for some reason they were hard-coded into the original.  @samm82, can you check whether this same mistake was propagated to Drasil?  If so, can you fix it there as well.

When you have had a chance to look into the Drasil status of this issue, please close this issue.  

I am also going to create another issue related to this because when I looked at the bib file I found that the existing entries are not proper.

**Comment by samm82** [comment]
My original intent with this issue was that there wasn't a section referenced after "Section.", but your change is a better fix - I found more examples of hardcoded references while looking through Drasil - I'll look for them in caseStudies, but I'm kind of stuck on the `! Undefined control sequence.` error in `\tableofcontents` in the build version of GlassBR as described in JacquesCarette/Drasil#701, as I can't verify my changes.

**Comment by samm82** [comment]
Waiting on JacquesCarette/Drasil#721.

**Comment by samm82** [comment]
Should this:
https://github.com/JacquesCarette/Drasil/blob/0324a081966f5555ca8bcd457d88ec287b8f0540/code/Language/Drasil/Chunk/Concept.hs#L41-L43

take a `Sentence` instead of a `String`? Then `foldlSent` could be used like this:
```haskell
annealedGl    = cc annealedGlass
  (foldlSent [S "A flat, monolithic, glass lite which has uniform thickness where the",
    S "residual surface stresses are almost zero, as defined in", cite gbRefDB astm2016)
```

instead of:
https://github.com/JacquesCarette/Drasil/blob/0324a081966f5555ca8bcd457d88ec287b8f0540/code/Example/Drasil/GlassBR/Unitals.hs#L257-L259

If so, this should probably be a separate issue, as it will be quite large/time intensive. @JacquesCarette @szymczdm @smiths

**Comment by smiths** [comment]
@samm82, maybe you should close this issue here, since the original problem with the manual case study has been fixed.  You can then create another issue in the Drasil repo if there are challenges for implementation in the generated version.

**Comment by samm82** [comment]
JacquesCarette/Drasil#742
